configure

```diff
@@ -1847,6 +1856,23 @@ emscripten)
 		exit 1
 	fi
 	;;
+kolibrios)
+	if test -z "$KOS32_SDK_DIR" || test -z "$KOS32_AUTOBUILD"; then
+		echo "Please set KOS32_SDK_DIR and KOS32_AUTOBUILD in your environment. export KOS32_SDK_DIR=<path to KOS32 SDK> and export KOS32_AUTOBUILD=<path to KOS32 build>"
+		exit 1
+	fi
+	_as="kos32-as"
+	_ar="kos32-ar cr"
+	_ranlib="kos32-ar -s"
+	_strip="kos32-strip"
+	if test -z "$CXX"; then
+		CXX="kos32-g++"
+	fi
+	append_var DEFINES "-U__WIN32__ -U_Win32 -U_WIN32 -U__MINGW32__ -UWIN32 -DKOLIBRIOS=1 -D_POSIX_C_SOURCE=1 -D_XOPEN_SOURCE=1"
+	append_var CXXFLAGS "-I${KOS32_SDK_DIR}/sources/newlib/libc/include -I${KOS32_SDK_DIR}/sources/zlib -I${KOS32_SDK_DIR}/sources/libstdc++-v3/include -fno-ident -fomit-frame-pointer"
+	append_var LDFLAGS "-nostdlib -shared -Wl,-shared -Wl,-s -Wl,-T${KOS32_SDK_DIR}/sources/newlib/dll.lds -Wl,--entry,_DllStartup -Wl,--image-base,0 -L${KOS32_SDK_DIR}/lib -L/home/autobuild/tools/win32/mingw32/lib -L/home/autobuild/tools/win32/lib"
+	append_var SYSLIBS "-ldll -lstdc++ -lsupc++ -lgcc -lc.dll -Dmain=xmain"
```

Comment:
So basically, kos32-g++ is not a proper C++ compiler...
I would try to add a `-specs=file` to CXXFLAGS. This is exactly the purpose of this file: specifying the system libraries needed.
Normally, GCC has the correct spec file bundled in.
More info on this [here](https://gcc.gnu.org/onlinedocs/gcc/Spec-Files.html).